tests/src/python/test_qgsaction.py

```diff
@@ -64,38 +64,41 @@ def _req_logger(self, params):
 
         self.assertEqual(self.body, br"a%26%2Bb=a%20and%20plus%20b&a%3Db=a%20equals%20b")
 
-    def test_post_multipart_action(self):
-        """Test multipart"""
-
-        self.body = None
-
-        def _req_logger(self, params):
-            self.body = bytes(params.content())
-
-        QgsNetworkAccessManager.instance().requestAboutToBeCreated[QgsNetworkRequestParameters].connect(partial(_req_logger, self))
-
-        temp_dir = QTemporaryDir()
-        temp_path = temp_dir.path()
-        temp_file = os.path.join(temp_path, 'multipart.txt')
-
-        action = QgsAction(QgsAction.SubmitUrlMultipart, 'url_encoded', "http://fake_qgis_http_endpoint" + temp_file + r"?[% url_encode(map('a&+b', 'a and plus b', 'a=b', 'a equals b')) %]")
-        ctx = QgsExpressionContext()
-        action.run(ctx)
-
-        while not self.body:
-            QgsApplication.instance().processEvents()
-
-        self.assertEqual(re.sub(r'\.oOo\.[^\r]*', '.oOo.UUID', self.body.decode('utf8')), '\r\n'.join([
-            '--boundary_.oOo.UUID',
-            'Content-Disposition: form-data; name="a&+b"',
-            '',
-            'a and plus b',
-            '--boundary_.oOo.UUID',
-            'Content-Disposition: form-data; name="a=b"',
-            '',
-            'a equals b',
-            '--boundary_.oOo.UUID',
-            '']))
+# TEST DISABLED
```

Comment:
I think you can remove the whole test case: I don't really know how to test it without the content: any idea?

I don't want to add a subprocess with a (python) server, this proved to be fragile in the past.

Reply:
Done.
I think it would be possible to add a test too by storing an action result inside the action, but I do not have the intention to go further in this area myself.